src/views/authenticated/List.tsx

```diff
@@ -16,17 +19,47 @@ export function List({ data: unfilteredListItems }: Props) {
 		);
 	}, [searchTerm, unfilteredListItems]);
 
+	const handleAddItemsClick = () => {
+		navigate("/manage-list");
+	};
+
 	return (
 		<>
 			<p>
 				Hello from the <code>/list</code> page!
 			</p>
 
+			{unfilteredListItems.length === 0 && (
+				<div>
+					<h2>Your list is ready!</h2>
+					<h3>
+						It looks like you haven’t added any items yet.
+						<br />
+						Let’s get started by adding your first item.
```

Comment:
thought: It may be itersting to think if we could make a shorter promot (so maybe more of a nitpick I'm sorry! 😅) but it maybe like:

"No items? Let's add the first one!"

...Well that felt cheesy, I am sorry for how unsmooth that was lol 😂

Reply:
I can see where you are coming from! We meant for the prompts to be conversational in tone and helpfully encouraging.
I actually agree though, that this prompt could be shorter:

You haven't added any items yet.
br
Let's get started by adding your first item!

Reply:
I align with this refactor as well! :point_up: